Reduce Citus Disk Size (#9796)
hashgraph/hedera-mirror-node: docs/runbook/reduce-disksize-citus.md

## Problem

Citus disks are over provisioned and need to be reduced in size.

## Prerequisites

> - All Citus PVCS are defined are sized in Gi

Reviewer: ```suggestion
- All Citus PVCS are defined in GiB
```

- `jq` is installed

> - The kubectl context is set to the cluster you want to reduce disk size for from

Reviewer: ```suggestion
- The kubectl context is set to the cluster with the over-sized disk
```
